File: app/src/main/java/com/brainwellnessspa/ResourceModule/Models/ResourceListModel.java
package com.brainwellnessspa.ResourceModule.Models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ResourceListModel {
    @SerializedName("ResponseData")
    @Expose
    private List<ResponseData> responseData = null;
    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseMessage")
    @Expose
    private String responseMessage;
    @SerializedName("ResponseStatus")
    @Expose
    private String responseStatus;

    public List<ResponseData> getResponseData() {
        return responseData;
    }

    public void setResponseData(List<ResponseData> responseData) {
        this.responseData = responseData;
    }

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public String getResponseMessage() {
        return responseMessage;
    }

    public void setResponseMessage(String responseMessage) {
        this.responseMessage = responseMessage;
    }

    public String getResponseStatus() {
        return responseStatus;
    }

    public void setResponseStatus(String responseStatus) {
        this.responseStatus = responseStatus;
    }

    public class ResponseData {
        @SerializedName("ID")
        @Expose
        private String iD;
        @SerializedName("title")
        @Expose
        private String title;
        @SerializedName("type")
        @Expose
        private String type;
        @SerializedName("master_category")
        @Expose
        private String masterCategory;
        @SerializedName("sub_category")
        @Expose
        private String subCategory;
        @SerializedName("ResourceDesc")
        @Expose
        private String description;
        @SerializedName("image")
        @Expose
        private String image;
        @SerializedName("author")
        @Expose
        private String author;
        @SerializedName("resource_link_1")
        @Expose
        private String resourceLink1;
        @SerializedName("resource_link_2")
        @Expose
        private String resourceLink2;

        public String getID() {
            return iD;
        }

        public void setID(String iD) {
            this.iD = iD;
        }

        public String getTitle() {
            return title;
        }

        public void setTitle(String title) {
            this.title = title;
        }

        public String getType() {
            return type;
        }

        public void setType(String type) {
            this.type = type;
        }

        public String getMasterCategory() {
            return masterCategory;
        }

        public void setMasterCategory(String masterCategory) {
            this.masterCategory = masterCategory;
        }

        public String getSubCategory() {
            return subCategory;
        }

        public void setSubCategory(String subCategory) {
            this.subCategory = subCategory;
        }

        public String getDescription() {
            return description;
        }

        public void setDescription(String description) {
            this.description = description;
        }

        public String getImage() {
            return image;
        }

        public void setImage(String image) {
            this.image = image;
        }

        public String getAuthor() {
            return author;
        }

        public void setAuthor(String author) {
            this.author = author;
        }

        public String getResourceLink1() {
            return resourceLink1;
        }

        public void setResourceLink1(String resourceLink1) {
            this.resourceLink1 = resourceLink1;
        }

        public String getResourceLink2() {
            return resourceLink2;
        }

        public void setResourceLink2(String resourceLink2) {
            this.resourceLink2 = resourceLink2;
        }
    }
}

File: app/src/main/java/com/brainwellnessspa/resourceModule/Models/ResourceListModel.kt
package com.brainwellnessspa.resourceModule.Models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ResourceListModel {
    @SerializedName("ResponseData")
    @Expose
    var responseData: List<ResponseData>? = null

    @SerializedName("ResponseCode")
    @Expose
    var responseCode: String? = null

    @SerializedName("ResponseMessage")
    @Expose
    var responseMessage: String? = null

    @SerializedName("ResponseStatus")
    @Expose
    var responseStatus: String? = null

    inner class ResponseData {
        @SerializedName("ID")
        @Expose
        var iD: String? = null

        @SerializedName("title")
        @Expose
        var title: String? = null

        @SerializedName("type")
        @Expose
        var type: String? = null

        @SerializedName("master_category")
        @Expose
        var masterCategory: String? = null

        @SerializedName("sub_category")
        @Expose
        var subCategory: String? = null

        @SerializedName("ResourceDesc")
        @Expose
        var description: String? = null

        @SerializedName("Detailimage")
        @Expose
        var detailimage: String? = null

        @SerializedName("image")
        @Expose
        var image: String? = null

        @SerializedName("author")
        @Expose
        var author: String? = null

        @SerializedName("resource_link_1")
        @Expose
        var resourceLink1: String? = null

        @SerializedName("resource_link_2")
        @Expose
        var resourceLink2: String? = null
    }
}